disable_prelinking.cf

```diff
@@ -1,35 +1,32 @@
 # Policy to disable prelinking
-
-bundle edit_line option_set_no(option)
-{
-  # Set option value to 'no'
-  replace_patterns:
-      "^$(option)=(?!no).*$"
-        replace_with => value("$(option)=no");
-}
-
 bundle agent disable_prelinking
 {
   # Enable autorun
   meta:
+    linux::
```

Comment:
Why this?

Reply:
prelink is a Linux program, right?
> On Linux, prelinking is accomplished via the prelink program, a free program written by Jakub Jelínek of Red Hat for ELF binaries. 
 - https://en.wikipedia.org/wiki/Prelink

Reply:
> Why this?

Because windows exists and this policy will error. I frequently forget to scope things down for windows.